[2022-11-15 15:58:50,256] {processor.py:153} INFO - Started process (PID=125) to work on /opt/airflow/dags/sensor.py
[2022-11-15 15:58:50,264] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 15:58:50,271] {logging_mixin.py:115} INFO - [2022-11-15 15:58:50,271] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 15:58:53,390] {logging_mixin.py:115} INFO - [2022-11-15 15:58:53,375] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 74, in <module>
    'path': dest_file_path,
NameError: name 'dest_file_path' is not defined
[2022-11-15 15:58:53,393] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 15:58:53,513] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.319 seconds
[2022-11-15 15:58:56,524] {processor.py:153} INFO - Started process (PID=130) to work on /opt/airflow/dags/sensor.py
[2022-11-15 15:58:56,535] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 15:58:56,546] {logging_mixin.py:115} INFO - [2022-11-15 15:58:56,545] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 15:58:58,398] {logging_mixin.py:115} INFO - [2022-11-15 15:58:58,383] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 74, in <module>
    'path': dest_file_path,
NameError: name 'dest_file_path' is not defined
[2022-11-15 15:58:58,403] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 15:58:58,505] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.024 seconds
[2022-11-15 15:59:13,682] {processor.py:153} INFO - Started process (PID=148) to work on /opt/airflow/dags/sensor.py
[2022-11-15 15:59:13,692] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 15:59:13,699] {logging_mixin.py:115} INFO - [2022-11-15 15:59:13,699] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 15:59:13,750] {logging_mixin.py:115} INFO - [2022-11-15 15:59:13,735] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 26
    def DownloadAndRenameFile(bucket_name:str1 path:str):
                                                  ^
SyntaxError: invalid syntax
[2022-11-15 15:59:13,761] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 15:59:15,974] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.389 seconds
[2022-11-15 15:59:18,996] {processor.py:153} INFO - Started process (PID=150) to work on /opt/airflow/dags/sensor.py
[2022-11-15 15:59:18,999] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 15:59:19,003] {logging_mixin.py:115} INFO - [2022-11-15 15:59:19,002] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 15:59:19,030] {logging_mixin.py:115} INFO - [2022-11-15 15:59:19,023] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 8
    import
         ^
SyntaxError: invalid syntax
[2022-11-15 15:59:19,033] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 15:59:19,558] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.609 seconds
[2022-11-15 15:59:27,240] {processor.py:153} INFO - Started process (PID=168) to work on /opt/airflow/dags/sensor.py
[2022-11-15 15:59:27,249] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 15:59:27,258] {logging_mixin.py:115} INFO - [2022-11-15 15:59:27,258] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 15:59:29,113] {logging_mixin.py:115} INFO - [2022-11-15 15:59:29,096] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 74, in <module>
    'path': dest_file_path,
NameError: name 'dest_file_path' is not defined
[2022-11-15 15:59:29,119] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 15:59:29,520] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.360 seconds
[2022-11-15 16:00:00,262] {processor.py:153} INFO - Started process (PID=205) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:00,274] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:00,285] {logging_mixin.py:115} INFO - [2022-11-15 16:00:00,283] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:00,927] {logging_mixin.py:115} INFO - [2022-11-15 16:00:00,923] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 74, in <module>
    'path': dest_file_path,
NameError: name 'dest_file_path' is not defined
[2022-11-15 16:00:00,929] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:00,973] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.725 seconds
[2022-11-15 16:00:06,722] {processor.py:153} INFO - Started process (PID=207) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:06,729] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:06,732] {logging_mixin.py:115} INFO - [2022-11-15 16:00:06,732] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:07,533] {logging_mixin.py:115} INFO - [2022-11-15 16:00:07,513] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 74, in <module>
    'path': dest_file_path,
NameError: name 'dest_file_path' is not defined
[2022-11-15 16:00:07,540] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:07,651] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.940 seconds
[2022-11-15 16:00:10,459] {processor.py:153} INFO - Started process (PID=228) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:10,461] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:10,469] {logging_mixin.py:115} INFO - [2022-11-15 16:00:10,469] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:11,014] {logging_mixin.py:115} INFO - [2022-11-15 16:00:11,010] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 74, in <module>
    'path': dest_file_path,
NameError: name 'dest_file_path' is not defined
[2022-11-15 16:00:11,017] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:11,053] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.602 seconds
[2022-11-15 16:00:15,110] {processor.py:153} INFO - Started process (PID=230) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:15,113] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:15,115] {logging_mixin.py:115} INFO - [2022-11-15 16:00:15,114] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:15,625] {logging_mixin.py:115} INFO - [2022-11-15 16:00:15,622] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 85, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:00:15,627] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:15,664] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.560 seconds
[2022-11-15 16:00:24,767] {processor.py:153} INFO - Started process (PID=248) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:24,769] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:24,771] {logging_mixin.py:115} INFO - [2022-11-15 16:00:24,771] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:25,380] {logging_mixin.py:115} INFO - [2022-11-15 16:00:25,377] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 85, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:00:25,385] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:25,436] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.677 seconds
[2022-11-15 16:00:28,203] {processor.py:153} INFO - Started process (PID=250) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:28,205] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:28,207] {logging_mixin.py:115} INFO - [2022-11-15 16:00:28,206] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:28,691] {logging_mixin.py:115} INFO - [2022-11-15 16:00:28,687] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 85, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:00:28,693] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:28,742] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.547 seconds
[2022-11-15 16:00:57,961] {processor.py:153} INFO - Started process (PID=288) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:00:57,963] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:00:57,966] {logging_mixin.py:115} INFO - [2022-11-15 16:00:57,965] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:58,521] {logging_mixin.py:115} INFO - [2022-11-15 16:00:58,517] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 103, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:00:58,523] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:00:58,558] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.604 seconds
[2022-11-15 16:01:10,682] {processor.py:153} INFO - Started process (PID=306) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:01:10,688] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:01:10,690] {logging_mixin.py:115} INFO - [2022-11-15 16:01:10,689] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:01:11,217] {logging_mixin.py:115} INFO - [2022-11-15 16:01:11,213] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 104, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:01:11,218] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:01:11,259] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.585 seconds
[2022-11-15 16:01:16,338] {processor.py:153} INFO - Started process (PID=311) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:01:16,340] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:01:16,343] {logging_mixin.py:115} INFO - [2022-11-15 16:01:16,342] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:01:18,659] {logging_mixin.py:115} INFO - [2022-11-15 16:01:18,646] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:01:18,661] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:01:18,700] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.374 seconds
[2022-11-15 16:01:48,965] {processor.py:153} INFO - Started process (PID=365) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:01:48,968] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:01:48,970] {logging_mixin.py:115} INFO - [2022-11-15 16:01:48,970] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:01:50,851] {logging_mixin.py:115} INFO - [2022-11-15 16:01:50,845] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:01:50,854] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:01:50,905] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.950 seconds
[2022-11-15 16:02:21,132] {processor.py:153} INFO - Started process (PID=404) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:02:21,133] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:02:21,135] {logging_mixin.py:115} INFO - [2022-11-15 16:02:21,135] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:02:22,790] {logging_mixin.py:115} INFO - [2022-11-15 16:02:22,786] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:02:22,792] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:02:22,828] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.705 seconds
[2022-11-15 16:02:40,866] {processor.py:153} INFO - Started process (PID=423) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:02:40,889] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:02:40,892] {logging_mixin.py:115} INFO - [2022-11-15 16:02:40,892] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:02:42,800] {logging_mixin.py:115} INFO - [2022-11-15 16:02:42,795] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:02:42,802] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:02:42,844] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.000 seconds
[2022-11-15 16:03:13,071] {processor.py:153} INFO - Started process (PID=478) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:03:13,073] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:03:13,076] {logging_mixin.py:115} INFO - [2022-11-15 16:03:13,076] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:03:14,871] {logging_mixin.py:115} INFO - [2022-11-15 16:03:14,865] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:03:14,873] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:03:14,910] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.848 seconds
[2022-11-15 16:03:38,730] {processor.py:153} INFO - Started process (PID=500) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:03:38,736] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:03:38,743] {logging_mixin.py:115} INFO - [2022-11-15 16:03:38,742] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:03:40,626] {logging_mixin.py:115} INFO - [2022-11-15 16:03:40,620] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:03:40,628] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:03:40,667] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.948 seconds
[2022-11-15 16:03:54,872] {processor.py:153} INFO - Started process (PID=535) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:03:54,874] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:03:54,876] {logging_mixin.py:115} INFO - [2022-11-15 16:03:54,876] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:03:57,112] {logging_mixin.py:115} INFO - [2022-11-15 16:03:57,098] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:03:57,116] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:03:57,181] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.316 seconds
[2022-11-15 16:04:01,419] {processor.py:153} INFO - Started process (PID=541) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:04:01,422] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:04:01,425] {logging_mixin.py:115} INFO - [2022-11-15 16:04:01,424] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:04:03,493] {logging_mixin.py:115} INFO - [2022-11-15 16:04:03,484] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:04:03,497] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:04:03,557] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.151 seconds
[2022-11-15 16:04:15,754] {processor.py:153} INFO - Started process (PID=560) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:04:15,756] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:04:15,760] {logging_mixin.py:115} INFO - [2022-11-15 16:04:15,759] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:04:17,798] {logging_mixin.py:115} INFO - [2022-11-15 16:04:17,793] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:04:17,801] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:04:17,848] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.108 seconds
[2022-11-15 16:04:32,863] {processor.py:153} INFO - Started process (PID=582) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:04:32,865] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:04:32,867] {logging_mixin.py:115} INFO - [2022-11-15 16:04:32,867] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:04:34,934] {logging_mixin.py:115} INFO - [2022-11-15 16:04:34,926] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 105, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:04:34,937] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:04:34,995] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.141 seconds
[2022-11-15 16:05:01,302] {processor.py:153} INFO - Started process (PID=617) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:05:01,305] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:05:01,308] {logging_mixin.py:115} INFO - [2022-11-15 16:05:01,308] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:04,717] {logging_mixin.py:115} INFO - [2022-11-15 16:05:04,681] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 107, in <module>
    python_callable=LoadAndUploadNewPrecios,
NameError: name 'LoadAndUploadNewPrecios' is not defined
[2022-11-15 16:05:04,722] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:04,930] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.641 seconds
[2022-11-15 16:05:21,273] {processor.py:153} INFO - Started process (PID=654) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:05:21,276] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:05:21,280] {logging_mixin.py:115} INFO - [2022-11-15 16:05:21,279] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:23,257] {logging_mixin.py:115} INFO - [2022-11-15 16:05:23,250] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:05:23,259] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:23,306] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.063 seconds
[2022-11-15 16:05:36,357] {processor.py:153} INFO - Started process (PID=673) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:05:36,360] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:05:36,363] {logging_mixin.py:115} INFO - [2022-11-15 16:05:36,362] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:38,179] {logging_mixin.py:115} INFO - [2022-11-15 16:05:38,172] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:05:38,182] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:38,234] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.895 seconds
[2022-11-15 16:05:47,521] {processor.py:153} INFO - Started process (PID=679) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:05:47,523] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:05:47,525] {logging_mixin.py:115} INFO - [2022-11-15 16:05:47,525] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:48,975] {logging_mixin.py:115} INFO - [2022-11-15 16:05:48,964] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:05:48,978] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:49,024] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.512 seconds
[2022-11-15 16:05:52,051] {processor.py:153} INFO - Started process (PID=698) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:05:52,054] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:05:52,057] {logging_mixin.py:115} INFO - [2022-11-15 16:05:52,056] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:52,088] {logging_mixin.py:115} INFO - [2022-11-15 16:05:52,084] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 124
    python_callable=ListarArchivos}
                                  ^
SyntaxError: invalid syntax
[2022-11-15 16:05:52,091] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:52,478] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 0.439 seconds
[2022-11-15 16:05:56,132] {processor.py:153} INFO - Started process (PID=700) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:05:56,144] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:05:56,146] {logging_mixin.py:115} INFO - [2022-11-15 16:05:56,146] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:57,567] {logging_mixin.py:115} INFO - [2022-11-15 16:05:57,560] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:05:57,569] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:05:57,618] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.503 seconds
[2022-11-15 16:06:09,636] {processor.py:153} INFO - Started process (PID=719) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:06:09,638] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:06:09,640] {logging_mixin.py:115} INFO - [2022-11-15 16:06:09,640] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:06:11,205] {logging_mixin.py:115} INFO - [2022-11-15 16:06:11,194] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:06:11,209] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:06:11,298] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.670 seconds
[2022-11-15 16:06:12,756] {processor.py:153} INFO - Started process (PID=725) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:06:12,758] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:06:12,760] {logging_mixin.py:115} INFO - [2022-11-15 16:06:12,760] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:06:14,395] {logging_mixin.py:115} INFO - [2022-11-15 16:06:14,390] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:06:14,397] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:06:14,430] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.683 seconds
[2022-11-15 16:06:44,722] {processor.py:153} INFO - Started process (PID=763) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:06:44,725] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:06:44,727] {logging_mixin.py:115} INFO - [2022-11-15 16:06:44,727] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:06:46,441] {logging_mixin.py:115} INFO - [2022-11-15 16:06:46,434] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:06:46,444] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:06:46,486] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.774 seconds
[2022-11-15 16:07:16,843] {processor.py:153} INFO - Started process (PID=817) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:07:16,846] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:07:16,848] {logging_mixin.py:115} INFO - [2022-11-15 16:07:16,848] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:07:18,447] {logging_mixin.py:115} INFO - [2022-11-15 16:07:18,419] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:07:18,451] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:07:18,502] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.679 seconds
[2022-11-15 16:07:48,754] {processor.py:153} INFO - Started process (PID=853) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:07:48,756] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:07:48,758] {logging_mixin.py:115} INFO - [2022-11-15 16:07:48,758] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:07:50,610] {logging_mixin.py:115} INFO - [2022-11-15 16:07:50,595] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:07:50,614] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:07:50,713] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.968 seconds
[2022-11-15 16:08:20,981] {processor.py:153} INFO - Started process (PID=891) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:08:20,986] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:08:20,995] {logging_mixin.py:115} INFO - [2022-11-15 16:08:20,994] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:08:27,836] {logging_mixin.py:115} INFO - [2022-11-15 16:08:27,822] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:08:27,839] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:08:28,060] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 7.128 seconds
[2022-11-15 16:08:58,768] {processor.py:153} INFO - Started process (PID=946) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:08:58,771] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:08:58,774] {logging_mixin.py:115} INFO - [2022-11-15 16:08:58,773] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:09:00,155] {logging_mixin.py:115} INFO - [2022-11-15 16:09:00,147] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:09:00,159] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:09:00,205] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.796 seconds
[2022-11-15 16:09:30,815] {processor.py:153} INFO - Started process (PID=984) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:09:30,819] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:09:30,821] {logging_mixin.py:115} INFO - [2022-11-15 16:09:30,821] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:09:32,029] {logging_mixin.py:115} INFO - [2022-11-15 16:09:32,024] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:09:32,031] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:09:32,067] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.547 seconds
[2022-11-15 16:10:03,043] {processor.py:153} INFO - Started process (PID=1022) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:10:03,046] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:10:03,048] {logging_mixin.py:115} INFO - [2022-11-15 16:10:03,047] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:10:04,349] {logging_mixin.py:115} INFO - [2022-11-15 16:10:04,344] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:10:04,351] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:10:04,389] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.632 seconds
[2022-11-15 16:10:34,852] {processor.py:153} INFO - Started process (PID=1060) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:10:34,855] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:10:34,857] {logging_mixin.py:115} INFO - [2022-11-15 16:10:34,856] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:10:36,070] {logging_mixin.py:115} INFO - [2022-11-15 16:10:36,065] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:10:36,072] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:10:36,126] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.518 seconds
[2022-11-15 16:11:06,606] {processor.py:153} INFO - Started process (PID=1114) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:11:06,610] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:11:06,612] {logging_mixin.py:115} INFO - [2022-11-15 16:11:06,611] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:11:07,769] {logging_mixin.py:115} INFO - [2022-11-15 16:11:07,762] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:11:07,772] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:11:07,813] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.460 seconds
[2022-11-15 16:11:39,061] {processor.py:153} INFO - Started process (PID=1153) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:11:39,064] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:11:39,066] {logging_mixin.py:115} INFO - [2022-11-15 16:11:39,065] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:11:40,230] {logging_mixin.py:115} INFO - [2022-11-15 16:11:40,224] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:11:40,232] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:11:40,269] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.549 seconds
[2022-11-15 16:12:10,796] {processor.py:153} INFO - Started process (PID=1191) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:12:10,799] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:12:10,801] {logging_mixin.py:115} INFO - [2022-11-15 16:12:10,801] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:12:12,068] {logging_mixin.py:115} INFO - [2022-11-15 16:12:12,061] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:12:12,070] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:12:12,108] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.558 seconds
[2022-11-15 16:12:42,704] {processor.py:153} INFO - Started process (PID=1239) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:12:42,707] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:12:42,709] {logging_mixin.py:115} INFO - [2022-11-15 16:12:42,709] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:12:44,179] {logging_mixin.py:115} INFO - [2022-11-15 16:12:44,158] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:12:44,189] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:12:44,252] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.887 seconds
[2022-11-15 16:13:15,018] {processor.py:153} INFO - Started process (PID=1311) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:13:15,023] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:13:15,026] {logging_mixin.py:115} INFO - [2022-11-15 16:13:15,026] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:13:16,574] {logging_mixin.py:115} INFO - [2022-11-15 16:13:16,560] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:13:16,577] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:13:16,640] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.992 seconds
[2022-11-15 16:13:47,200] {processor.py:153} INFO - Started process (PID=1349) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:13:47,203] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:13:47,205] {logging_mixin.py:115} INFO - [2022-11-15 16:13:47,205] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:13:48,449] {logging_mixin.py:115} INFO - [2022-11-15 16:13:48,444] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:13:48,451] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:13:48,639] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.697 seconds
[2022-11-15 16:14:19,295] {processor.py:153} INFO - Started process (PID=1387) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:14:19,312] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:14:19,325] {logging_mixin.py:115} INFO - [2022-11-15 16:14:19,324] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:14:21,919] {logging_mixin.py:115} INFO - [2022-11-15 16:14:21,907] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:14:21,922] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:14:22,027] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.757 seconds
[2022-11-15 16:14:52,569] {processor.py:153} INFO - Started process (PID=1432) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:14:52,577] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:14:52,580] {logging_mixin.py:115} INFO - [2022-11-15 16:14:52,580] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:14:55,323] {logging_mixin.py:115} INFO - [2022-11-15 16:14:55,302] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:14:55,329] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:14:55,509] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.959 seconds
[2022-11-15 16:15:25,983] {processor.py:153} INFO - Started process (PID=1476) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:15:25,985] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:15:25,987] {logging_mixin.py:115} INFO - [2022-11-15 16:15:25,987] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:15:27,333] {logging_mixin.py:115} INFO - [2022-11-15 16:15:27,328] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:15:27,335] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:15:27,373] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.398 seconds
[2022-11-15 16:15:57,688] {processor.py:153} INFO - Started process (PID=1514) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:15:57,691] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:15:57,693] {logging_mixin.py:115} INFO - [2022-11-15 16:15:57,693] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:15:58,791] {logging_mixin.py:115} INFO - [2022-11-15 16:15:58,786] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:15:58,793] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:15:58,830] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.150 seconds
[2022-11-15 16:16:29,098] {processor.py:153} INFO - Started process (PID=1552) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:16:29,105] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:16:29,107] {logging_mixin.py:115} INFO - [2022-11-15 16:16:29,107] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:16:30,253] {logging_mixin.py:115} INFO - [2022-11-15 16:16:30,247] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:16:30,256] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:16:30,293] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.205 seconds
[2022-11-15 16:17:00,495] {processor.py:153} INFO - Started process (PID=1590) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:17:00,498] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:17:00,499] {logging_mixin.py:115} INFO - [2022-11-15 16:17:00,499] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:17:01,693] {logging_mixin.py:115} INFO - [2022-11-15 16:17:01,679] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:17:01,695] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:17:01,741] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.253 seconds
[2022-11-15 16:17:31,982] {processor.py:153} INFO - Started process (PID=1638) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:17:31,989] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:17:31,996] {logging_mixin.py:115} INFO - [2022-11-15 16:17:31,996] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:17:33,916] {logging_mixin.py:115} INFO - [2022-11-15 16:17:33,897] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:17:33,920] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:17:33,992] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.022 seconds
[2022-11-15 16:18:04,254] {processor.py:153} INFO - Started process (PID=1682) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:18:04,255] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:18:04,257] {logging_mixin.py:115} INFO - [2022-11-15 16:18:04,257] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:18:05,433] {logging_mixin.py:115} INFO - [2022-11-15 16:18:05,428] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:18:05,436] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:18:05,477] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.231 seconds
[2022-11-15 16:18:35,800] {processor.py:153} INFO - Started process (PID=1720) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:18:35,802] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:18:35,804] {logging_mixin.py:115} INFO - [2022-11-15 16:18:35,804] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:18:36,982] {logging_mixin.py:115} INFO - [2022-11-15 16:18:36,973] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:18:36,984] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:18:37,036] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.250 seconds
[2022-11-15 16:19:07,311] {processor.py:153} INFO - Started process (PID=1759) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:19:07,313] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:19:07,315] {logging_mixin.py:115} INFO - [2022-11-15 16:19:07,315] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:19:08,661] {logging_mixin.py:115} INFO - [2022-11-15 16:19:08,651] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:19:08,663] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:19:08,718] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.418 seconds
[2022-11-15 16:19:38,891] {processor.py:153} INFO - Started process (PID=1797) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:19:38,896] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:19:38,899] {logging_mixin.py:115} INFO - [2022-11-15 16:19:38,899] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:19:40,327] {logging_mixin.py:115} INFO - [2022-11-15 16:19:40,312] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:19:40,330] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:19:40,420] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.542 seconds
[2022-11-15 16:20:10,757] {processor.py:153} INFO - Started process (PID=1850) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:20:10,760] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:20:10,763] {logging_mixin.py:115} INFO - [2022-11-15 16:20:10,762] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:20:12,050] {logging_mixin.py:115} INFO - [2022-11-15 16:20:12,044] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:20:12,052] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:20:12,102] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.353 seconds
[2022-11-15 16:20:42,299] {processor.py:153} INFO - Started process (PID=1892) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:20:42,305] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:20:42,308] {logging_mixin.py:115} INFO - [2022-11-15 16:20:42,307] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:20:43,554] {logging_mixin.py:115} INFO - [2022-11-15 16:20:43,548] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 12, in <module>
    import findspark
ModuleNotFoundError: No module named 'findspark'
[2022-11-15 16:20:43,556] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:20:43,598] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 1.308 seconds
[2022-11-15 16:27:07,907] {processor.py:153} INFO - Started process (PID=128) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:27:07,916] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:27:07,930] {logging_mixin.py:115} INFO - [2022-11-15 16:27:07,929] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:28:48,447] {processor.py:153} INFO - Started process (PID=111) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:28:48,462] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:28:48,465] {logging_mixin.py:115} INFO - [2022-11-15 16:28:48,464] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:29:14,808] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:29:17,023] {logging_mixin.py:115} INFO - [2022-11-15 16:29:17,022] {manager.py:508} INFO - Created Permission View: can read on DAG:DAG_Minio_S3_Wait_for_File
[2022-11-15 16:29:17,100] {logging_mixin.py:115} INFO - [2022-11-15 16:29:17,099] {manager.py:508} INFO - Created Permission View: can delete on DAG:DAG_Minio_S3_Wait_for_File
[2022-11-15 16:29:17,134] {logging_mixin.py:115} INFO - [2022-11-15 16:29:17,134] {manager.py:508} INFO - Created Permission View: can edit on DAG:DAG_Minio_S3_Wait_for_File
[2022-11-15 16:29:17,137] {logging_mixin.py:115} INFO - [2022-11-15 16:29:17,136] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:29:17,198] {logging_mixin.py:115} INFO - [2022-11-15 16:29:17,195] {dag.py:2439} INFO - Creating ORM DAG for DAG_Minio_S3_Wait_for_File
[2022-11-15 16:29:17,313] {logging_mixin.py:115} INFO - [2022-11-15 16:29:17,312] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-10T00:00:00+00:00, run_after=2022-11-11T00:00:00+00:00
[2022-11-15 16:29:17,408] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 28.988 seconds
[2022-11-15 16:29:47,771] {processor.py:153} INFO - Started process (PID=168) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:29:47,776] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:29:47,783] {logging_mixin.py:115} INFO - [2022-11-15 16:29:47,782] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:29:50,916] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:29:51,070] {logging_mixin.py:115} INFO - [2022-11-15 16:29:51,068] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:29:51,141] {logging_mixin.py:115} INFO - [2022-11-15 16:29:51,141] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-10T00:00:00+00:00, run_after=2022-11-11T00:00:00+00:00
[2022-11-15 16:29:51,173] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.473 seconds
[2022-11-15 16:30:21,537] {processor.py:153} INFO - Started process (PID=206) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:30:21,540] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:30:21,543] {logging_mixin.py:115} INFO - [2022-11-15 16:30:21,542] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:30:24,471] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:30:24,655] {logging_mixin.py:115} INFO - [2022-11-15 16:30:24,649] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:30:24,749] {logging_mixin.py:115} INFO - [2022-11-15 16:30:24,749] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-10T00:00:00+00:00, run_after=2022-11-11T00:00:00+00:00
[2022-11-15 16:30:24,784] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.263 seconds
[2022-11-15 16:30:55,275] {processor.py:153} INFO - Started process (PID=253) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:30:55,309] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:30:55,313] {logging_mixin.py:115} INFO - [2022-11-15 16:30:55,313] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:31:25,374] {logging_mixin.py:115} INFO - [2022-11-15 16:31:25,344] {timeout.py:67} ERROR - Process timed out, PID: 253
[2022-11-15 16:31:25,418] {logging_mixin.py:115} INFO - [2022-11-15 16:31:25,384] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 31, in <module>
    from airflow.providers.amazon.aws.sensors.s3 import S3KeySensor
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/amazon/aws/sensors/s3.py", line 37, in <module>
    from airflow.providers.amazon.aws.hooks.s3 import S3Hook
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/amazon/aws/hooks/s3.py", line 35, in <module>
    from boto3.s3.transfer import S3Transfer, TransferConfig
  File "/home/airflow/.local/lib/python3.7/site-packages/boto3/s3/transfer.py", line 129, in <module>
    from s3transfer.futures import NonThreadedExecutor
  File "/home/airflow/.local/lib/python3.7/site-packages/s3transfer/futures.py", line 14, in <module>
    import logging
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 68, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/sensor.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#reducing-dag-complexity, PID: 253
[2022-11-15 16:31:25,428] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:31:25,619] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 30.421 seconds
[2022-11-15 16:31:56,838] {processor.py:153} INFO - Started process (PID=311) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:31:56,841] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:31:56,850] {logging_mixin.py:115} INFO - [2022-11-15 16:31:56,849] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:31:59,999] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:32:00,224] {logging_mixin.py:115} INFO - [2022-11-15 16:32:00,222] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:32:00,319] {logging_mixin.py:115} INFO - [2022-11-15 16:32:00,319] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:32:00,348] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.529 seconds
[2022-11-15 16:32:30,698] {processor.py:153} INFO - Started process (PID=350) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:32:30,700] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:32:30,702] {logging_mixin.py:115} INFO - [2022-11-15 16:32:30,702] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:32:33,489] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:32:33,684] {logging_mixin.py:115} INFO - [2022-11-15 16:32:33,682] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:32:33,790] {logging_mixin.py:115} INFO - [2022-11-15 16:32:33,787] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:32:33,913] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.223 seconds
[2022-11-15 16:33:04,296] {processor.py:153} INFO - Started process (PID=388) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:33:04,298] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:33:04,302] {logging_mixin.py:115} INFO - [2022-11-15 16:33:04,302] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:33:07,008] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:33:07,328] {logging_mixin.py:115} INFO - [2022-11-15 16:33:07,316] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:33:07,558] {logging_mixin.py:115} INFO - [2022-11-15 16:33:07,558] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:33:07,650] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.369 seconds
[2022-11-15 16:33:38,940] {processor.py:153} INFO - Started process (PID=425) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:33:38,944] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:33:38,950] {logging_mixin.py:115} INFO - [2022-11-15 16:33:38,949] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:33:41,443] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:33:41,582] {logging_mixin.py:115} INFO - [2022-11-15 16:33:41,581] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:33:41,659] {logging_mixin.py:115} INFO - [2022-11-15 16:33:41,659] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:33:41,686] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.756 seconds
[2022-11-15 16:34:12,031] {processor.py:153} INFO - Started process (PID=462) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:34:12,033] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:34:12,036] {logging_mixin.py:115} INFO - [2022-11-15 16:34:12,035] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:34:15,728] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:34:15,884] {logging_mixin.py:115} INFO - [2022-11-15 16:34:15,882] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:34:15,965] {logging_mixin.py:115} INFO - [2022-11-15 16:34:15,965] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:34:15,993] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.975 seconds
[2022-11-15 16:34:46,487] {processor.py:153} INFO - Started process (PID=501) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:34:46,490] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:34:46,492] {logging_mixin.py:115} INFO - [2022-11-15 16:34:46,492] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:34:48,788] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:34:48,939] {logging_mixin.py:115} INFO - [2022-11-15 16:34:48,937] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:34:49,045] {logging_mixin.py:115} INFO - [2022-11-15 16:34:49,044] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:34:49,081] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.603 seconds
[2022-11-15 16:35:19,395] {processor.py:153} INFO - Started process (PID=554) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:35:19,397] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:35:19,400] {logging_mixin.py:115} INFO - [2022-11-15 16:35:19,399] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:35:21,871] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:35:22,034] {logging_mixin.py:115} INFO - [2022-11-15 16:35:22,032] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:35:22,230] {logging_mixin.py:115} INFO - [2022-11-15 16:35:22,229] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:35:22,329] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.943 seconds
[2022-11-15 16:35:52,875] {processor.py:153} INFO - Started process (PID=592) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:35:52,877] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:35:52,879] {logging_mixin.py:115} INFO - [2022-11-15 16:35:52,879] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:35:55,510] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:35:55,691] {logging_mixin.py:115} INFO - [2022-11-15 16:35:55,688] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:35:55,796] {logging_mixin.py:115} INFO - [2022-11-15 16:35:55,795] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:35:55,841] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.977 seconds
[2022-11-15 16:36:26,391] {processor.py:153} INFO - Started process (PID=630) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:36:26,395] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:36:26,398] {logging_mixin.py:115} INFO - [2022-11-15 16:36:26,397] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:36:29,664] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:36:29,941] {logging_mixin.py:115} INFO - [2022-11-15 16:36:29,938] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:36:30,052] {logging_mixin.py:115} INFO - [2022-11-15 16:36:30,051] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:36:30,115] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.743 seconds
[2022-11-15 16:37:00,624] {processor.py:153} INFO - Started process (PID=667) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:37:00,627] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:37:00,632] {logging_mixin.py:115} INFO - [2022-11-15 16:37:00,631] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:37:03,232] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:37:03,395] {logging_mixin.py:115} INFO - [2022-11-15 16:37:03,392] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:37:03,489] {logging_mixin.py:115} INFO - [2022-11-15 16:37:03,489] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:37:03,521] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.907 seconds
[2022-11-15 16:37:33,809] {processor.py:153} INFO - Started process (PID=721) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:37:33,811] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:37:33,814] {logging_mixin.py:115} INFO - [2022-11-15 16:37:33,813] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:37:36,187] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:37:36,300] {logging_mixin.py:115} INFO - [2022-11-15 16:37:36,299] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:37:36,372] {logging_mixin.py:115} INFO - [2022-11-15 16:37:36,372] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:37:36,398] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.598 seconds
[2022-11-15 16:38:06,669] {processor.py:153} INFO - Started process (PID=759) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:38:06,672] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:38:06,675] {logging_mixin.py:115} INFO - [2022-11-15 16:38:06,674] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:38:09,305] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:38:09,460] {logging_mixin.py:115} INFO - [2022-11-15 16:38:09,458] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:38:09,607] {logging_mixin.py:115} INFO - [2022-11-15 16:38:09,606] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:38:09,670] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.013 seconds
[2022-11-15 16:38:39,964] {processor.py:153} INFO - Started process (PID=797) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:38:39,967] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:38:39,971] {logging_mixin.py:115} INFO - [2022-11-15 16:38:39,970] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:38:43,352] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:38:43,531] {logging_mixin.py:115} INFO - [2022-11-15 16:38:43,529] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:38:43,612] {logging_mixin.py:115} INFO - [2022-11-15 16:38:43,611] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:38:43,640] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.687 seconds
[2022-11-15 16:39:13,865] {processor.py:153} INFO - Started process (PID=850) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:39:13,868] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:39:13,870] {logging_mixin.py:115} INFO - [2022-11-15 16:39:13,870] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:39:16,887] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:39:17,034] {logging_mixin.py:115} INFO - [2022-11-15 16:39:17,032] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:39:17,109] {logging_mixin.py:115} INFO - [2022-11-15 16:39:17,109] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:39:17,141] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.286 seconds
[2022-11-15 16:39:47,330] {processor.py:153} INFO - Started process (PID=889) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:39:47,335] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:39:47,337] {logging_mixin.py:115} INFO - [2022-11-15 16:39:47,337] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:39:49,385] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:39:49,530] {logging_mixin.py:115} INFO - [2022-11-15 16:39:49,529] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:39:49,609] {logging_mixin.py:115} INFO - [2022-11-15 16:39:49,609] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:39:49,636] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.315 seconds
[2022-11-15 16:40:19,882] {processor.py:153} INFO - Started process (PID=927) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:40:19,885] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:40:19,887] {logging_mixin.py:115} INFO - [2022-11-15 16:40:19,886] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:40:22,282] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:40:22,458] {logging_mixin.py:115} INFO - [2022-11-15 16:40:22,455] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:40:22,576] {logging_mixin.py:115} INFO - [2022-11-15 16:40:22,576] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:40:22,703] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.830 seconds
[2022-11-15 16:40:53,195] {processor.py:153} INFO - Started process (PID=965) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:40:53,202] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:40:53,222] {logging_mixin.py:115} INFO - [2022-11-15 16:40:53,207] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:41:02,277] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:41:02,501] {logging_mixin.py:115} INFO - [2022-11-15 16:41:02,499] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:41:02,588] {logging_mixin.py:115} INFO - [2022-11-15 16:41:02,587] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:41:02,618] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 9.478 seconds
[2022-11-15 16:41:33,186] {processor.py:153} INFO - Started process (PID=1026) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:41:33,193] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:41:33,205] {logging_mixin.py:115} INFO - [2022-11-15 16:41:33,203] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:42:03,489] {logging_mixin.py:115} INFO - [2022-11-15 16:42:03,362] {timeout.py:67} ERROR - Process timed out, PID: 1026
[2022-11-15 16:42:06,914] {logging_mixin.py:115} INFO - [2022-11-15 16:42:04,463] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 10, in <module>
    import pandas as pd
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/__init__.py", line 50, in <module>
    from pandas.core.api import (
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/api.py", line 29, in <module>
    from pandas.core.arrays import Categorical
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/arrays/__init__.py", line 20, in <module>
    from pandas.core.arrays.string_arrow import ArrowStringArray
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/arrays/string_arrow.py", line 65, in <module>
    import pyarrow.compute as pc
  File "/home/airflow/.local/lib/python3.7/site-packages/pyarrow/compute.py", line 251, in <module>
    _make_global_functions()
  File "/home/airflow/.local/lib/python3.7/site-packages/pyarrow/compute.py", line 248, in _make_global_functions
    g[cpp_name] = g[name] = _wrap_function(name, func)
  File "/home/airflow/.local/lib/python3.7/site-packages/pyarrow/compute.py", line 227, in _wrap_function
    return _decorate_compute_function(wrapper, name, func, option_class)
  File "/home/airflow/.local/lib/python3.7/site-packages/pyarrow/compute.py", line 146, in _decorate_compute_function
    wrapper.__doc__ = "".join(dedent(s) for s in doc_pieces)
  File "/home/airflow/.local/lib/python3.7/site-packages/pyarrow/compute.py", line 146, in <genexpr>
    wrapper.__doc__ = "".join(dedent(s) for s in doc_pieces)
  File "/usr/local/lib/python3.7/textwrap.py", line 461, in dedent
    text = re.sub(r'(?m)^' + margin, '', text)
  File "/usr/local/lib/python3.7/re.py", line 194, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 68, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/sensor.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#reducing-dag-complexity, PID: 1026
[2022-11-15 16:42:07,913] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:42:26,957] {processor.py:153} INFO - Started process (PID=1059) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:42:26,995] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:42:27,055] {logging_mixin.py:115} INFO - [2022-11-15 16:42:27,043] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:42:56,918] {logging_mixin.py:115} INFO - [2022-11-15 16:42:56,887] {timeout.py:67} ERROR - Process timed out, PID: 1059
[2022-11-15 16:42:56,974] {logging_mixin.py:115} INFO - [2022-11-15 16:42:56,928] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 10, in <module>
    import pandas as pd
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/__init__.py", line 50, in <module>
    from pandas.core.api import (
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/api.py", line 48, in <module>
    from pandas.core.groupby import (
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/groupby/__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import (
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/groupby/generic.py", line 73, in <module>
    from pandas.core.frame import DataFrame
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 963, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 906, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1280, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1252, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1391, in find_spec
  File "<frozen importlib._bootstrap_external>", line 59, in _path_join
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 68, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/sensor.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#reducing-dag-complexity, PID: 1059
[2022-11-15 16:42:56,990] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:42:57,608] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 31.300 seconds
[2022-11-15 16:43:29,758] {processor.py:153} INFO - Started process (PID=1124) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:43:29,773] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:43:29,784] {logging_mixin.py:115} INFO - [2022-11-15 16:43:29,784] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:43:59,871] {logging_mixin.py:115} INFO - [2022-11-15 16:43:59,833] {timeout.py:67} ERROR - Process timed out, PID: 1124
[2022-11-15 16:43:59,966] {logging_mixin.py:115} INFO - [2022-11-15 16:43:59,904] {dagbag.py:321} ERROR - Failed to import: /opt/airflow/dags/sensor.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 318, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/sensor.py", line 13, in <module>
    from pyspark import SparkContext
  File "/home/airflow/.local/lib/python3.7/site-packages/pyspark/__init__.py", line 64, in <module>
    from pyspark.version import __version__
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 670, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 589, in module_from_spec
  File "<frozen importlib._bootstrap>", line 568, in _init_module_attrs
  File "<frozen importlib._bootstrap>", line 409, in cached
  File "<frozen importlib._bootstrap_external>", line 372, in _get_cached
  File "<frozen importlib._bootstrap_external>", line 308, in cache_from_source
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 68, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/sensor.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.3.3/best-practices.html#reducing-dag-complexity, PID: 1124
[2022-11-15 16:43:59,975] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:44:00,645] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 30.995 seconds
[2022-11-15 16:44:32,938] {processor.py:153} INFO - Started process (PID=1177) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:44:32,943] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:44:32,962] {logging_mixin.py:115} INFO - [2022-11-15 16:44:32,961] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:44:39,153] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:44:39,446] {logging_mixin.py:115} INFO - [2022-11-15 16:44:39,444] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:44:39,553] {logging_mixin.py:115} INFO - [2022-11-15 16:44:39,552] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:44:39,608] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 6.728 seconds
[2022-11-15 16:45:09,962] {processor.py:153} INFO - Started process (PID=1230) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:45:09,966] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:45:09,968] {logging_mixin.py:115} INFO - [2022-11-15 16:45:09,968] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:45:12,717] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:45:12,901] {logging_mixin.py:115} INFO - [2022-11-15 16:45:12,899] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:45:12,988] {logging_mixin.py:115} INFO - [2022-11-15 16:45:12,988] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:45:13,032] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.087 seconds
[2022-11-15 16:45:43,708] {processor.py:153} INFO - Started process (PID=1268) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:45:43,711] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:45:43,715] {logging_mixin.py:115} INFO - [2022-11-15 16:45:43,715] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:45:49,587] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:45:49,777] {logging_mixin.py:115} INFO - [2022-11-15 16:45:49,773] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:45:49,869] {logging_mixin.py:115} INFO - [2022-11-15 16:45:49,868] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:45:49,907] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 6.233 seconds
[2022-11-15 16:46:20,213] {processor.py:153} INFO - Started process (PID=1306) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:46:20,217] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:46:20,221] {logging_mixin.py:115} INFO - [2022-11-15 16:46:20,221] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:46:22,824] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:46:22,996] {logging_mixin.py:115} INFO - [2022-11-15 16:46:22,994] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:46:23,095] {logging_mixin.py:115} INFO - [2022-11-15 16:46:23,094] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:46:23,129] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.925 seconds
[2022-11-15 16:46:53,712] {processor.py:153} INFO - Started process (PID=1344) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:46:53,716] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:46:53,720] {logging_mixin.py:115} INFO - [2022-11-15 16:46:53,720] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:46:58,483] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:46:58,694] {logging_mixin.py:115} INFO - [2022-11-15 16:46:58,692] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:46:58,773] {logging_mixin.py:115} INFO - [2022-11-15 16:46:58,773] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:46:58,807] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 5.119 seconds
[2022-11-15 16:47:29,102] {processor.py:153} INFO - Started process (PID=1398) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:47:29,104] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:47:29,106] {logging_mixin.py:115} INFO - [2022-11-15 16:47:29,105] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:47:31,373] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:47:31,510] {logging_mixin.py:115} INFO - [2022-11-15 16:47:31,509] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:47:31,591] {logging_mixin.py:115} INFO - [2022-11-15 16:47:31,590] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:47:31,619] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.527 seconds
[2022-11-15 16:48:02,424] {processor.py:153} INFO - Started process (PID=1437) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:48:02,427] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:48:02,429] {logging_mixin.py:115} INFO - [2022-11-15 16:48:02,429] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:48:04,964] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:48:05,137] {logging_mixin.py:115} INFO - [2022-11-15 16:48:05,136] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:48:05,241] {logging_mixin.py:115} INFO - [2022-11-15 16:48:05,241] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:48:05,277] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.865 seconds
[2022-11-15 16:48:35,510] {processor.py:153} INFO - Started process (PID=1475) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:48:35,512] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:48:35,514] {logging_mixin.py:115} INFO - [2022-11-15 16:48:35,514] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:48:37,306] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:48:37,413] {logging_mixin.py:115} INFO - [2022-11-15 16:48:37,412] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:48:37,482] {logging_mixin.py:115} INFO - [2022-11-15 16:48:37,481] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:48:37,505] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.001 seconds
[2022-11-15 16:49:07,791] {processor.py:153} INFO - Started process (PID=1530) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:49:07,793] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:49:07,795] {logging_mixin.py:115} INFO - [2022-11-15 16:49:07,795] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:49:09,898] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:49:10,012] {logging_mixin.py:115} INFO - [2022-11-15 16:49:10,011] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:49:10,082] {logging_mixin.py:115} INFO - [2022-11-15 16:49:10,081] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:49:10,107] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.325 seconds
[2022-11-15 16:49:40,240] {processor.py:153} INFO - Started process (PID=1568) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:49:40,243] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:49:40,247] {logging_mixin.py:115} INFO - [2022-11-15 16:49:40,247] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:49:42,350] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:49:42,476] {logging_mixin.py:115} INFO - [2022-11-15 16:49:42,475] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:49:42,544] {logging_mixin.py:115} INFO - [2022-11-15 16:49:42,543] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:49:42,568] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.340 seconds
[2022-11-15 16:50:13,528] {processor.py:153} INFO - Started process (PID=1606) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:50:13,530] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:50:13,532] {logging_mixin.py:115} INFO - [2022-11-15 16:50:13,532] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:50:15,502] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:50:15,619] {logging_mixin.py:115} INFO - [2022-11-15 16:50:15,617] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:50:15,689] {logging_mixin.py:115} INFO - [2022-11-15 16:50:15,689] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:50:15,717] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.198 seconds
[2022-11-15 16:50:45,904] {processor.py:153} INFO - Started process (PID=1654) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:50:45,907] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:50:45,909] {logging_mixin.py:115} INFO - [2022-11-15 16:50:45,909] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:50:48,755] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:50:49,010] {logging_mixin.py:115} INFO - [2022-11-15 16:50:49,007] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:50:49,113] {logging_mixin.py:115} INFO - [2022-11-15 16:50:49,112] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:50:49,152] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.260 seconds
[2022-11-15 16:51:20,153] {processor.py:153} INFO - Started process (PID=1698) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:51:20,155] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:51:20,157] {logging_mixin.py:115} INFO - [2022-11-15 16:51:20,157] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:51:22,595] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:51:22,772] {logging_mixin.py:115} INFO - [2022-11-15 16:51:22,770] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:51:22,866] {logging_mixin.py:115} INFO - [2022-11-15 16:51:22,865] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:51:22,902] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 2.760 seconds
[2022-11-15 16:51:53,251] {processor.py:153} INFO - Started process (PID=1736) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:51:53,256] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:51:53,259] {logging_mixin.py:115} INFO - [2022-11-15 16:51:53,259] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:51:56,208] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:51:56,353] {logging_mixin.py:115} INFO - [2022-11-15 16:51:56,351] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:51:56,439] {logging_mixin.py:115} INFO - [2022-11-15 16:51:56,438] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:51:56,468] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 3.235 seconds
[2022-11-15 16:52:26,905] {processor.py:153} INFO - Started process (PID=1790) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:52:26,907] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:52:26,915] {logging_mixin.py:115} INFO - [2022-11-15 16:52:26,913] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:52:31,742] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:52:31,990] {logging_mixin.py:115} INFO - [2022-11-15 16:52:31,988] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:52:32,081] {logging_mixin.py:115} INFO - [2022-11-15 16:52:32,080] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:52:32,114] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 5.227 seconds
[2022-11-15 16:53:02,687] {processor.py:153} INFO - Started process (PID=1827) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:53:02,691] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:53:02,694] {logging_mixin.py:115} INFO - [2022-11-15 16:53:02,693] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:53:08,584] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:53:08,798] {logging_mixin.py:115} INFO - [2022-11-15 16:53:08,796] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:53:08,890] {logging_mixin.py:115} INFO - [2022-11-15 16:53:08,889] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:53:08,932] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 6.263 seconds
[2022-11-15 16:53:39,476] {processor.py:153} INFO - Started process (PID=1869) to work on /opt/airflow/dags/sensor.py
[2022-11-15 16:53:39,488] {processor.py:641} INFO - Processing file /opt/airflow/dags/sensor.py for tasks to queue
[2022-11-15 16:53:39,493] {logging_mixin.py:115} INFO - [2022-11-15 16:53:39,493] {dagbag.py:508} INFO - Filling up the DagBag from /opt/airflow/dags/sensor.py
[2022-11-15 16:53:46,252] {processor.py:651} INFO - DAG(s) dict_keys(['DAG_Minio_S3_Wait_for_File']) retrieved from /opt/airflow/dags/sensor.py
[2022-11-15 16:53:46,874] {logging_mixin.py:115} INFO - [2022-11-15 16:53:46,869] {dag.py:2420} INFO - Sync 1 DAGs
[2022-11-15 16:53:46,995] {logging_mixin.py:115} INFO - [2022-11-15 16:53:46,994] {dag.py:2972} INFO - Setting next_dagrun for DAG_Minio_S3_Wait_for_File to 2022-11-15T00:00:00+00:00, run_after=2022-11-16T00:00:00+00:00
[2022-11-15 16:53:47,036] {processor.py:161} INFO - Processing /opt/airflow/dags/sensor.py took 7.575 seconds
